client/packages/system/src/Patient/api/api.ts

```diff
@@ -0,0 +1,72 @@
+import { SortBy, NameSortFieldInput } from '@openmsupply-client/common';
+import { Sdk, NameRowFragment } from './operations.generated';
+
+export type ListParams = {
+  first?: number;
+  offset?: number;
+  sortBy?: SortBy<NameRowFragment>;
+};
+
+export const getPatientQueries = (sdk: Sdk, storeId: string) => ({
+  get: {
+    byId: async (nameId: string) => {
+      const result = await sdk.nameById({ storeId, nameId });
+      const { names } = result;
+      if (names.__typename === 'NameConnector') {
+        if (names.nodes.length) {
+          return names.nodes[0];
+        }
+      }
+
+      throw new Error('Name not found');
+    },
+    list: async ({
+      first,
+      offset,
+      sortBy,
+    }: ListParams): Promise<{
+      nodes: NameRowFragment[];
+      totalCount: number;
+    }> => {
+      const key =
+        sortBy?.key === 'name'
+          ? NameSortFieldInput.Name
+          : NameSortFieldInput.Code;
+
+      const result = await sdk.names({
+        first,
+        offset,
+        key,
+        desc: !!sortBy?.isDesc,
+        storeId,
+        filter: {
+          isCustomer: true,
+        },
+      });
```

Comment:
Should we make this `isCustomer: false`?

Reply:
when I created the PR, there was no patient filter ( still isn't ) and no type to distinguish patients.
This is a temporary filter, to get some results to display